### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by DannyBen** [marionebl/svg-term-cli#96]
There seem to be no new commits, and there are several issues.

If it is abandoned, perhaps a notice on the README is appropriate?

**Comment by unphased** [comment]
I havent dug into this project yet, and it looks like I won't, but I'm doing a little shallow dive to figure out the best way to generate and share terminal screencasts. I like the svg concept but I think one thing that gives me pause from this format already is lack of time scrubber. 

embedding an h.264 video for example is probably fine in practice, however outside of OSs that perform OCR for you (which is unreliable) you cannot copy text content out of that. I think time scrubbing is more important than being able to copy so that is the direction i'm going in.

**Comment by co2water** [comment]
Looks like it, last release 2024 and 48 open issues. If you need something maintained: https://github.com/co2water/svgcast. Same idea (asciicast to animated svg), handles long recordings without the heap OOM from this tracker, and does dark mode from one file.

**Comment by unphased** [comment]
your project looks great! i wonder if svg could ever support a time scrubber. Not needed for short demos but it would solve a lot of problems for much longer recordings. I have to say though it is quite a compelling thing to be able to drop an efficient terminal recording as an svg in a page.

**Comment by co2water** [comment]
Fair ask, and I looked at it properly: a readme image is an `<img>`, no script runs and no clicks get through, and github's raw urls add a sandbox csp on top. So a scrubber inside the readme itself is not possible for anyone.

What I could do, in the next release: `--progress` draws a thin bar along the bottom (pure css, so it does animate inside the readme), and `--controls` adds click-to-seek on that bar, click to pause, space and arrow keys. `--controls` only works where the svg is opened as a document, your own site, pages, an `<object>`. So: bar in the readme, scrubber when you link to the file. Ships in v0.2.0.

**Comment by vorburger** [comment]
https://github.com/asciinema/asciinema-player is what I've [switched to](https://github.com/enola-dev/enola/commit/e9a71fee60c115e3c892c0aa271f6bca05490cf1). 

It's great of your goal is having screen casts embedded on HTML pages in high quality.

**Comment by co2water** [comment]
Agreed, if you own the page asciinema-player is the better answer. Real scrubber, real player, and it solves what this thread originally asked for.

svgcast is for the one place it can't go: a readme, or anywhere that only takes an `<img>`. Same .cast feeds both, so my setup is the player on the docs site and the svg in the readme.
